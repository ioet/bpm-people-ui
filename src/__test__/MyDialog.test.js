import React from 'react';
import BpmDialog from '../component/presentational/BpmDialog';

describe('<BpmDialog />', () => {
  test('Component Renders correctly', () => {
    const appWrapper = shallow(<BpmDialog />);

    expect(toJson(appWrapper))
      .toMatchSnapshot();
  });
});
