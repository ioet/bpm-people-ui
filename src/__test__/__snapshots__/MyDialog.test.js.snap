// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyDialog /> Component Renders correctly 1`] = `
<WithStyles(Dialog)
  TransitionComponent={[Function]}
  keepMounted={true}
  onClose={[Function]}
>
  <WithStyles(DialogTitle) />
  <WithStyles(DialogContent) />
  <WithStyles(DialogActions)>
    <WithStyles(Button)
      color="primary"
      onClick={[Function]}
    >
      Cancel
    </WithStyles(Button)>
    <WithStyles(Button)
      color="primary"
      onClick={[Function]}
    />
  </WithStyles(DialogActions)>
</WithStyles(Dialog)>
`;
